flang/test/Lower/entry-statement.f90

```diff
@@ -67,6 +67,7 @@ subroutine ss(n1)
   ! CHECK: fir.alloca i32 {{{.*}}uniq_name = "{{.*}}Enx"}
   ! CHECK: fir.alloca i32 {{{.*}}uniq_name = "{{.*}}Eny"}
   integer n17, n2
+  procedure(), pointer :: procptr1
```

Comment:
This code should hit a TODO, we should not silently compile and do nothing with the procedure pointer, even if it is unused here.

Reply:
I've reverted the change.